client/snapshot.go

```diff
@@ -100,6 +104,7 @@ func (sh *Snapshot) ContentHash() ([]byte, error) {
 	sh2.SetID = 0
 	sh2.Time = time.Time{}
 	sh2.Auto = false
+	sh2.Options = nil
```

Comment:
@pedronis , @mvo5 , please just double check that you agree that snapshot options should not be part of calculating snapshot hash/uniqueness.

Reply:
This was approved by @pedronis in specification.